odxtools/flashclass.py

```diff
@@ -0,0 +1,32 @@
+# SPDX-License-Identifier: MIT
+from dataclasses import dataclass
+from typing import Any
+from xml.etree import ElementTree
+
+from .element import IdentifiableElement
+from .odxdoccontext import OdxDocContext
+from .odxlink import OdxLinkDatabase, OdxLinkId
+from .snrefcontext import SnRefContext
+from .utils import dataclass_fields_asdict
+
+
+@dataclass(kw_only=True)
+class FlashClass(IdentifiableElement):
+    """
+    Corresponds to FLASH-CLASS.
+    """
```

Comment:
No properties?

Reply:
nope, but the spec has this tag type, so we should have it as well. (ADDITIONAL-AUDIENCE is the same, BTW.)